docs/LoongArch-ELF-ABI-CN.adoc

```diff
@@ -647,6 +647,205 @@ v1.00
 |R_LARCH_GNU_VTENTRY
 |GNU C++ vtable 支持
 |
+
+4+|... 保留项
+
+|64
+|R_LARCH_B16
+|18 位相对 PC 跳转
+|`+(*(uint32_t *) PC) [25 ... 10] = (S+A-PC) [17 ... 2]+`
+
+带 18 位有符号数溢出和4字节对齐检测功能
+
+|65
+|R_LARCH_B21
+|23 位相对 PC 跳转
+|`+(*(uint32_t *) PC) [4 ... 0] = (S+A-PC) [22 ... 18],+`
+
+`+(*(uint32_t *) PC) [25 ... 10] = (S+A-PC) [17 ... 2]+`
+
+带 23 位有符号数溢出和4字节对齐检测功能
+
+|66
+|R_LARCH_B26
+|28 位相对 PC 跳转
+|`+(*(uint32_t *) PC) [9 ... 0] = (S+A-PC) [27 ... 18],+`
+
+`+(*(uint32_t *) PC) [25 ... 10] = (S+A-PC) [17 ... 2]+`
+
+带 28 位有符号数溢出和4字节对齐检测功能
+
+|67
+|R_LARCH_ABS_HI20
+|32/64 位绝对地址的 [31 ... 12] 位
+|`+(*(uint32_t *) PC) [24 ... 5] = (S+A) [31 ... 12]+`
+
+|68
+|R_LARCH_ABS_LO12
+|32/64 位绝对地址的  [11 ... 0]  位
+|`+(*(uint32_t *) PC) [21 ... 10] = (S+A) [11 ... 0]+`
+
+|69
+|R_LARCH_ABS64_LO20
+|64 位绝对地址 [51 ... 32] 位
+|`+(*(uint32_t *) PC) [24 ... 5] = (S+A) [51 ... 32]+`
+
+|70
+|R_LARCH_ABS64_HI12
+|64 位绝对地址 [63 ... 52] 位
+|`+(*(uint32_t *) PC) [21 ... 10] = (S+A) [63 ... 52]+`
+
+|71
+|R_LARCH_PCALA_HI20
+|相对 PC 偏移 32/64 位的 [31 ... 12] 位
+|`+(*(uint32_t *) PC) [24 ... 5] = (((S+A) & ~0xfff) - (PC & ~0xfff)) [31 ... 12]+`
+
+`+注意：所有相对 PC 偏移计算都不包含低12位。+`
+
+|72
+|R_LARCH_PCALA_LO12
+|32/64 位地址的 [11 ... 0] 位
+|`+(*(uint32_t *) PC) [21 ... 10] = (S+A) [11 ... 0]+`
+
+|73
+|R_LARCH_PCALA64_LO20
+|相对 PC 偏移 64 位的 [51 ... 32] 位
+|`+(*(uint32_t *) PC) [24 ... 5] = (S+A - (PC & ~0xffffffff)) [51 ... 32]+`
+
+|74
+|R_LARCH_PCALA64_HI12
+|相对 PC 偏移 64 位的 [63 ... 52] 位
+|`+(*(uint32_t *) PC) [21 ... 10] = (S+A - (PC & ~0xffffffff)) [63 ... 52]+`
+
+|75
+|R_LARCH_GOT_PC_HI20
+|GOT 表项相对 PC 偏移 32/64 位的 [31 ... 12] 位
+|`+(*(uint32_t *) PC) [24 ... 5] = (((GP+G) & ~0xfff) - (PC & ~0xfff)) [31 ... 12]+`
+
+|76
+|R_LARCH_GOT_PC_LO12
+|GOT 表项 32/64 位地址的 [11 ... 0] 位
+|`+(*(uint32_t *) PC) [21 ... 10] = (GP+G) [11 ... 0]+`
+
+|77
+|R_LARCH_GOT64_PC_LO20
+|GOT 表项相对 PC 偏移 64 位的 [51 ... 32] 位
+|`+(*(uint32_t *) PC) [24 ... 5] = (GP+G - (PC & ~0xffffffff)) [51 ... 32]+`
+
+|78
+|R_LARCH_GOT64_PC_HI12
+|GOT 表项相对 PC 偏移 64 位的 [63 ... 52] 位
+|`+(*(uint32_t *) PC) [21 ... 10] = (GP+G - (PC & ~0xffffffff)) [63 ... 52]+`
+
+|79
+|R_LARCH_GOT_HI20
+|GOT 表项 32/64 位绝对地址的 [31 ... 12] 位
+|`+(*(uint32_t *) PC) [24 ... 5] = (GP+G) [31 ... 12]+`
+
+|80
+|R_LARCH_GOT_LO12
+|GOT 表项 32/64 位绝对地址的 [11 ... 0] 位
+|`+(*(uint32_t *) PC) [21 ... 10] = (GP+G) [11 ... 0]+`
```

Comment:
The names of 79 and 80 are not the same as in the code, which is correct?

The names in the binutils code are:
```
R_LARCH_GOT64_HI20 79
R_LARCH_GOT64_LO12 80
```
https://github.com/loongson/binutils-gdb/blob/291cb57c2947868201be6c2b0b903d318a7df90d/include/elf/loongarch.h#L161

Reply:
Please see https://sourceware.org/pipermail/binutils/2022-July/121937.html